aerogear/data-sync-ui
# Aerogear 7579

JIRA https://issues.jboss.org/browse/AEROGEAR-7579

To test:

Create two data sources, one called a, one called b
Filter for a 
Create a new one called aa 
It should appear in list
